projects/ui-policy-studio-angular/src/lib/components/flow-form-dialog/flow-llm-form-dialog/gio-ps-flow-llm-form-dialog.component.scss

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@use 'sass:map';
+@use '@angular/material' as mat;
+@use '@gravitee/ui-particles-angular' as gio;
+
+$typography: map.get(gio.$mat-theme, typography);
+
+.title {
+  &__closeBtn {
+    top: 0;
+    right: -24px;
+    float: right;
+  }
+}
+
+.content {
+  &__paramTitle {
+    @include mat.m2-typography-level($typography, subtitle-2);
+
+    color: mat.m2-get-color-from-palette(gio.$mat-dove-palette, 'darker60');
+  }
+
+  &--params {
+    margin-bottom: 6px;
+  }
+
+  &__row {
+    display: flex;
+    align-items: flex-start;
+    gap: 16px;
+
+    &__fullWidth {
```

Comment:
Ok yes style are broken because you have copy past this form MCP scss component and copy past HTML form Http Proxy component.
Please could you clean scss and html to have something consistent. We expect to have the same display as for HTTP Proxy no ?